spec/serializers/api/internal/v1/offence_serializer_spec.rb

```diff
@@ -1,29 +1,42 @@
-# frozen_string_literal: true
-
 RSpec.describe Api::Internal::V1::OffenceSerializer do
-  subject { described_class.new(offence).serializable_hash }
-
-  let(:offence) do
-    instance_double("Offence",
-                    id: "UUID",
-                    code: "AA06001",
-                    order_index: "0",
-                    title: "Fail to wear protective clothing",
-                    legislation: "Offences against the Person Act 1861 s.24",
-                    mode_of_trial: "Indictable-Only Offence",
-                    mode_of_trial_reasons: [{ description: "Court directs trial by jury", code: "5" }],
-                    pleas: %w[GUILTY NOT_GUILTY])
+  let(:serialized_data) do
+    offence_data = JSON.parse(file_fixture("offence/all_fields.json").read)
+    offence = Offence.new(body: offence_data, details: [offence_data])
```

Comment:
Does the body here not need to come from offence_summary_data?

Reply:
yes, good catch! thank you